lib/plugins/physics.js

```diff
@@ -128,7 +134,7 @@ function inject(bot) {
 
     // limit speed
     var currentMaxGroundSpeed;
-    var underBlock=bot.blockAt(pos.offset(0,-1,0));
+    var underBlock=bot.blockAt(new Vec3(pos.x,Math.ceil(pos.y-1.002),pos.z));
```

Comment:
Yeah I might do that. I think it's possible 0.001 might be enough even.
The ceiling is needed because now it possible for pos.y to be 63+7/8, which correctly gives 64 with ceiling, but for normal blocks we sometimes have pos.y = 64.001 which should be ceiled to 64, not 65. 
Maybe a MARGIN_ABOVE_GROUND constant would make sense.